vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
the vendor directory should be added to the .gitignore file and not committed to this repo.